my_vulkan/interop_utils.cpp

```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+#include "interop_utils.hpp"
+
+std::vector<const char *> my_vulkan::get_vk_device_extensions_for_cuda(bool offscreen)
```

Comment:
hmm, these are not limited to cuda, ar they?

Reply:
I only tested interop with cuda. I do not know if interop with other things will require different extensions. Maybe some requires more some requires less. That is why I put the cuda in the name. This function only return sets of extensions. If we want to interop with other stuff such as opengl or opencl, we can always give them a special sets of extensions, `extensions_for_opencl` for example.